python/cuml/cuml/manifold/umap.pyx

```diff
@@ -395,7 +403,9 @@ class UMAP(UniversalBase,
         if init == "spectral" or init == "random":
             self.init = init
         else:
-            raise Exception("Initialization strategy not supported: %d" % init)
+            gs = GlobalSettings()
+            if not(gs.accelerator_active or self._experimental_dispatching):
```

Comment:
Formatting nit - `not` is not a function in python, should be

```suggestion
            if not (gs.accelerator_active or self._experimental_dispatching):
```

Our formatting linter doesn't run in cython unfortunately; one of the perks of moving most code out of cython to python is we could avoid spending time on nits like this :).